src/js/xr/src/hooks/ui-manager/index.js

```diff
@@ -80,6 +82,11 @@ for (let propertyName of ['intensity', 'penumbra']) {
   )
 }
 
+lenses.distance = R.lens(
+  vin => clamp(mapLinear(vin, 0, 100, 0, 1), 0, 1),
+  vout => clamp(steps(mapLinear(vout, 0, 1, 0, 100), 0.1), 0, 100)
```

Comment:
Is there a reason this doesn't match Shot Generator (which uses 0.025...100) ? https://github.com/wonderunit/storyboarder/blob/master/src/js/shot-generator/Components.js#L1211

Reply:
Tried dragging around in Shot Generator when committing this and was getting `0.1` increments! But if it's supposed to be 0.025 then let's change it.